docs/getting_started.md

```
@@ -14,38 +14,45 @@ This tutorial will provide a brief overview of:
 
 ## Installing required packages
 
-Download and install [SyncroSim version 2.2.19](https://syncrosim.com/download/){:target="_blank"} or higher and follow the installation prompts.
+Download and install [SyncroSim version 3.1.29](https://syncrosim.com/download/){:target="_blank"} or higher and follow the installation prompts.
 
-Open SyncroSim and select **File > Packages… > Install…**. Then, mark the check-boxes beside the **stsim** and **landfirevegmodels** packages and click OK.
+Open SyncroSim and select **File > Local Packages… > Install from Server**. Then, mark the check-boxes beside the **stsim** (version 4.5.3 or later).
 
-You may need to update the **stsim** package to version 3.2.21 or later. To do so, select **File > Packages > Update...**, mark the check-box beside the **stsim** package and click **OK**.
+<img align="middle" style="padding: 1px" width="500" src="assets/images/image-1.png">
```

markdownlint-cli2 (0.22.1)

[warning] Images should have alternate text (alt text)

(MD045, no-alt-text)